.github/CODEOWNERS

```diff
@@ -12,6 +12,14 @@
 /gems/stackprof         @pocke
 /gems/rails-dom-testing @ksss
 /gems/yard              @ksss
+/gems/ast            @pocke
+/gems/aws-sdk-*      @ksss
+/gems/business_time  @ksss
+/gems/dogstatsd-ruby @VTRyo
+/gems/faraday        @yykamei
+/gems/i18n           @takahashim
+/gems/rolify         @ksss
+/gems/stackprof      @pocke
```

Comment:
Could you fix this file to avoid duplication?

Reply:
fixed 👍